cou/utils/openstack.py

```diff
@@ -417,13 +417,15 @@ def lookup(cls, charm: str) -> dict:
 def is_charm_supported(charm: str) -> bool:
     """Check if a charm upgrade is supported.
 
+    Currently data plane apps are not supported.
     :param charm: Name of the charm.
     :type charm: str
     :return: True if supported, else False
     :rtype: bool
     """
     return (
-        bool(OpenStackCodenameLookup.lookup(charm))
+        charm not in DATA_PLANE_CHARMS
```

Comment:
Caw we add warning logging that charm is not supported?

Reply:
Like this we will raise and exception if charm `nova-compute` is not supported, but can we provide list of unsupported charms instead?

Reply:
Ah ok so there will no exception, in that is ok. But It would be nice to add warning logs for data plane charms. So user will see that these are not supported, not for each unsupported charm but only for data plane charms. I think it would be as good feedback to user.

Reply:
I think this work should be scope of another PR that address the issue of #105

Reply:
ACK